packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/e2e/quanticGeneratedAnswer.e2e.ts

```diff
@@ -232,6 +233,35 @@ useCaseTestCases.forEach((useCase) => {
                 expect(showMoreButtonLabel).toEqual(expectedShowMoreLabel);
               });
             });
+
+            if (config.options.answerConfigurationId) {
+              test.describe('when selecting a timeframe facet after the answer is generated', () => {
+                test.use({
+                  options: {
+                    ...config.options,
+                  },
+                  withFacets: true,
+                });
+                test('should trigger a new generate call to the answer API', async ({
+                  generatedAnswer,
+                }) => {
+                  await generatedAnswer.streamEndAnalyticRequestPromise;
+
+                  const generateRequestPromise =
+                    generatedAnswer.waitForGenerateRequest({
+                      q: exampleQuery,
+                    });
+                  await generatedAnswer.clickFirstTimeframeFacetLink();
+                  await generateRequestPromise;
+
+                  await generatedAnswer.cleanup();
```

Comment:
You already have this here, why do you also need it in the fixture?

Also which test was breaking?